src/chains/ethereum/ethereum/src/blockchain.ts

```diff
@@ -365,6 +370,11 @@ export default class Blockchain extends Emittery.Typed<
         );
       });
 
+      // save storage keys to the database one at a time
```

Comment:
```suggestion
      // save storage keys to the database
```

Because this loop happens within the `batch` callback, teeeecchhhhnically they'll all get saved at the same time :-)